### PR TITLE
Fix example in readme.md

### PR DESCRIPTION
**Pull request by johnacarvana**
CompleteResult has no parameters.

### CODE REVIEW

**Comment by AMalininHere** [comment]
Hello! Thank you for your PR!

File: README.md [hunk 1]
```diff
@@ -16,9 +16,12 @@ public class SayHelloHandler : IExternalTaskHandler
 
         await Task.Delay(1000);
 
-        return new CompleteResult(new Dictionary<string, Variable>
+        return Task.FromResult<IExecutionResult>(new CompleteResult
```
Comment (AMalininHere) [inline README.md:19]: `Task.FromResult<IExecutionResult>` is unnecessary in this case

**Comment by AMalininHere** [comment]
@johnacarvana could you look at my note?

**Comment by johnacarvana** [comment]
Apologies for the delay. Update pushed this morning.

**Comment by AMalininHere** [comment]
Thank you for your contribution!

**Pull request merged**